scapy/config.py

```diff
@@ -326,7 +326,9 @@ def __repr__(self):
         # type: () -> str
         s = []
         for li in sorted(self, key=lambda x: x.__name__):
-            doc = li.__doc__.split("\n")[0] if li.__doc__ else "--"
+            doc = li.__doc__ if li.__doc__ else "--"
+            doc = doc.lstrip('\n').lstrip()
+            doc = doc.split("\n")[0]
```

Comment:
```suggestion
            doc = doc.lstrip().split("\n", 1)[0]
```

IMO it's a bit cleaner to use only `.lstrip()` (I don't understand why `.lstrip("\n")` first). Also, it's better to specify a `maxsplit` value, since we only want to keep the first part (some linters even require it).

Reply:
Thanks. I did not know that `lstrip()`  removes `\n` and ` ` at the same time.

Reply:
FWIW, also strips `\t` and `\r` (all "spaces"). Same for `.split()`.